code/http4s/src/main/scala/com/avast/datadog4s/extension/http4s/MetricsOpsBuilder.scala

```diff
@@ -0,0 +1,39 @@
+package com.avast.datadog4s.extension.http4s
+
+import cats.effect.Sync
+import cats.effect.concurrent.Ref
+import com.avast.datadog4s.api.{ MetricFactory, Tag }
+import com.avast.datadog4s.extension.http4s.DatadogMetricsOps.ClassifierTags
+import com.avast.datadog4s.extension.http4s.MetricsOpsBuilder.defaultClassifierTags
+import com.avast.datadog4s.extension.http4s.impl.{ ActiveConnections, DefaultMetricsOps }
+import org.http4s.metrics.MetricsOps
+import cats.syntax.functor._
+
+private[http4s] final class MetricsOpsBuilder[F[_]: Sync](metricFactory: MetricFactory[F]) {
```

Comment:
I think you meant to make the constructor private and not the whole class, didn't you?

Reply:
🤦 yep